door_button.py

```diff
@@ -86,7 +88,12 @@ def button_pressed_action(meeting_providers, key_press_counter, message_player):
             message_player.queue_text(greet)
         for (name, provider) in meeting_providers:
             meeting = provider.get_current_meeting()
-            message_player.queue_text(get_availablilty_message(meeting, name))
+            if meeting is not None:
+                message_player.queue_text(get_availablilty_message(meeting, name))
+                meetingexists += 1
```

Comment:
Variable meetingexists have to be declared.
We use snake_case naming conversion so it should be `meeting_exists`

add a declaration 
`meeting_exists = False` before the loop 
then set `meeting_exists = True`